kubernetes/services/nginx-storage.yml.j2

```diff
@@ -38,6 +38,12 @@ spec:
         - name: nginx-storage-config
           configMap:
             name: nginx-storage-config
+      # securityContext allows you to add permission for pod and mounted volumes to specific group.
+      # We need this permission to create files and directories on mounted persistent disk.
+      # Group "65534" is the default group for user nobody in nginx-storage docker.
+      # User nobody is responsible for creating files and directories.
+      securityContext:
+        fsGroup: 65534
```

Comment:
How do you know it's 65534 and what makes you think that this ID will not change? Is the value guaranteed?

Reply:
This `65534` ID is commonly reserved for user `nobody` on mostly linux version. Lot of sources confirm that, example:
- https://www.debian.org/doc/debian-policy/#uid-and-gid-classes

- http://www.linfo.org/uid.html

- https://en.wikipedia.org/wiki/Group_identifier

Reply:
OK. Nice. You should have written it in the comment in the code.

Reply:
I have added it in a comment.